### ISSUE TITLE
"Getting bash completion" instructions are unclear

### ISSUE THREAD

**Issue by adrianludwin** [bazelbuild/bazel#1539]
The instructions for installing bash completion [say](http://www.bazel.io/docs/install.html):

> 1. Build it with Bazel: bazel build //scripts:bazel-complete.bash.

However, if I've just installed Bazel via the installer (or, I believe, through apt-get) it's far from clear to me what workspace I'm supposed to use. Can we clarify these instructions?

**Comment by davido** [comment]
Indeed, I guess something like that worked for me:

```
  $ grep -i bazel ~/.bashrc 
  # Bazel completion
  . /home/davido/.bazel/bin/bazel-complete.bash
```

**Comment by chbfiv** [comment]
bazel build //scripts:bazel-complete.bash.
.........................................................
ERROR: no such package 'scripts': BUILD file not found on package path.
INFO: Elapsed time: 0.335s

**Comment by kchodorow** [comment]
Updating the docs, but since it'll take a couple days to appear on the public site:

> Bazel comes with a bash completion script, which the installer copies into the
> `bin` directory. If you ran the installer with `--user`, this will be
> `$HOME/.bazel/bin`. If you ran the installer sudo, `/usr/local/bazel/bin`.
> 
> Copy the `bazel-complete.bash` script to your completion folder
> (`/etc/bash_completion.d` directory under Ubuntu). If you don't have a
> completion folder, you can copy it wherever suits you and simply insert
> `source /path/to/bazel-complete.bash` in your `~/.bashrc` file (under OS X, put
> it in your `~/.bash_profile` file).
> 
> If you built Bazel from source, the bash completion target is in the `//scripts`
> package:
> 
> 1. Build it with Bazel: `bazel build //scripts:bazel-complete.bash`.
> 2. Copy the script `bazel-bin/scripts/bazel-complete.bash` to one of the
>    locations described above.
>

**Comment by drcege** [comment]
I'm confusing if I installed  bazel via `custom APT repository` on Ubuntu, then how could I get `bazel-complete.bash` ?

**Comment by katre** [comment]
The debian/ubuntu package released by the Bazel team also installs the bash completion script in /etc.

**Comment by drcege** [comment]
@katre  I just installed it again using the recommended [Bazel custom APT repository](https://docs.bazel.build/versions/master/install-ubuntu.html#install-on-ubuntu), but I'm definitely sure there is no bazel completion script in /etc.

```
Ubuntu 16.04.2
bazel 0.5.0
```

**Comment by katre** [comment]
@drcege Try again with the latest released version, 0.5.0. There was a bug with the paths in debian packages around 0.5.0 which should be fixed now.

**Comment by drcege** [comment]
Oh, I find it! It's already in `/etc/bash_completion.d`, and named `bazel`.
I was looking for file `bazel-complete.bash`  under /etc directory. I mean the documentation is not so clear......
